### PR TITLE
fix(ffe-account-selector-react): Properly aligns reset and expand button

### PR DESCRIPTION
**Pull request by TorgeirHeim**
screenshot-1

### CODE REVIEW

**Comment by wkillerud** [comment]
Is this fix related to an issue? Please tag it if so :) 
Could you attach a screenshot of the end result?

File: packages/ffe-account-selector-react/less/base-selector.less [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
-        padding-right: 57px;
+        padding-right: 63px;
         &::-ms-clear {
             display: none;
         }
     }
 
     &__expand-button {
-        height: 23px;
-        width : 23px;
```
Comment (wkillerud) [inline packages/ffe-account-selector-react/less/base-selector.less:15]: I'm a bit wary of removing `height` and `width` - could you double-check that the component still looks good in IE?  ([free ready-made VMs from Microsoft with different IE versions can be downloaded here if you need them](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/))

**Review by wkillerud — CHANGES_REQUESTED** [review]
The changes are fine if the component still looks OK in IE, but please double-check :+1:

File: packages/ffe-account-selector-react/less/base-selector.less (hunk 1 above)
Comment (TorgeirHeim) [reply packages/ffe-account-selector-react/less/base-selector.less:15]: Verified in IE

**Review by wkillerud — APPROVED** [review]

**Pull request merged**